packages/google_sign_in/google_sign_in/ios/Classes/FLTGoogleSignInPlugin.m

```diff
@@ -112,6 +112,7 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
       result(error != nil ? getFlutterError(error) : @{
         @"idToken" : authentication.idToken,
         @"accessToken" : authentication.accessToken,
+        @"serverAuthCode" : currentUser.serverAuthCode,
```

Comment:
When app is started with an older Google Sign In state the `currentUser.serverAuthCode` can be nil and this will cause a crash since nil can't be inserted into NSDictionary

Reply:
That's right, I had this behaviour also.

Reply:
Hey guys, can you please detail how to reproduce this issue? We've tried a few scenarios with this branch, but none caused a crash. @Dahlgren

Reply:
It occurs when an existing session with Google Sign In is reused. The `serverAuthCode` only exists on initial login. If the existing session is flushed (logged out) and a new session is created the `serverAuthCode` will exist, if applicable.